src/pixie.c

```diff
@@ -18,8 +18,8 @@ static int msleep(long millisecs) {
 #include "send.h"
 #include <stdio.h>
 #include <stdlib.h>
-#include <string.h>
```

Comment:
this header is required for strcmp

Reply:
pixie.h already has.

Why not put the include in the .h file?

Reply:
because it's clearer.